lib/src/main/java/uk/gov/di/ipv/core/library/dto/CredentialIssuerConfig.java

```diff
@@ -25,27 +24,23 @@ public class CredentialIssuerConfig {
     private String jarEncryptionPublicJwk;
     private String audienceForClients;
     private URI ipvCoreRedirectUrl;
-    private String allowedSharedAttributes;
```

Comment:
We added this property in https://govukverify.atlassian.net/browse/PYIC-2566. We are planning to read all configuration from this path credentialIssuers/criId/connections. I have some questions about this changes. Will we keep to all CredentialIssuerConfig params in this path credentialIssuers/criId or not? We have duplicate keys and confusing in the future. Can we discuss it?

Reply:
Amrit and Shivanshu have a plan that keep some properties at the higher level not within connections. Next plan remove some properties with different PR at high level.